src/ORM/Table.php

```diff
@@ -968,6 +968,14 @@ public function findList(Query $query, array $options)
             trigger_error('Option "idField" is deprecated, use "keyField" instead.', E_USER_WARNING);
         }
 
+        if (!$query->clause('select')) {
+            $fields = array_merge((array)$options['keyField'], (array)$options['valueField']);
```

Comment:
Also, converting to array is a bad idea, it is possible for any of those values to be a closure, so that is going to give you a fatal error.